config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -264,6 +364,31 @@ periodics:
       - --timeout=500m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20181105-66d3bcb98-master
 
+- interval: 30m
+  name: ci-kubernetes-e2e-gci-gce-serial-kube-dns-nodecache
```

Comment:
Do we need `ci-kubernetes-e2e-gci-gce-serial-coredns-nodecache`?

Reply:
There is no gci-gce-serial-coredns

Reply:
Ah got it, thanks!